stylesheets/partials/core/_colors.scss

```diff
@@ -1,4 +1,5 @@
 /* Theme colors and accents */
+$turquoise-dark: #137875;
```

Comment:
what sam wanted here was the existing color $teal-dark: rgb(20,120,118); which is very close the the hex you used